packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -47,7 +47,7 @@ describe('Sequence - ', () => {
       }
     }
     // bind user defined sequence
-    app.bind('sequence').toClass(MySequence);
+    app.sequence(MySequence);
```

Comment:
I was +1 for interface too, but it looks like we're heading towards the extension path (some discussions during LBN-sync while you were away -- anyways, we chat more about during the next meeting).